app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1558,8 +1734,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
     }
     
     private void createPreferencesMenu() {
```

Comment:
Remove this private method, replace the once place it is called with the public version and 'this' as the input argument.